stacker/providers/aws/default.py

```diff
@@ -418,6 +420,7 @@ def generate_cloudformation_args(stack_name, parameters, tags, template,
         "Parameters": parameters,
         "Tags": tags,
         "Capabilities": capabilities,
+        "NotificationARNs": notification_arns
```

Comment:
Since this could be blank, you might want to follow the pattern below for optional arguments.  Not sure how boto handles a blank list for the argument here.